visallo/visallo
# Added: audit service to audit login, logout, and access denied messages

Testing Instructions:
- Login verify message in logs
- Logout verify message in logs
- Login and wait for session expire verify message in logs
- Try performing an action you don't have access to, verify message inlogs

CHANGELOG
Added: audit service to audit login, logout, and access denied messages
